platform/x11/detect.py

```diff
@@ -266,8 +266,8 @@ def configure(env):
         # mbedTLS does not provide a pkgconfig config yet. See https://github.com/ARMmbed/mbedtls/issues/228
         env.Append(LIBS=['mbedtls', 'mbedcrypto', 'mbedx509'])
 
-    if not env['builtin_libwebsockets']:
-        env.ParseConfig('pkg-config libwebsockets --cflags --libs')
+    if not env['builtin_wslay']:
+        env.ParseConfig('pkg-config libwslay --cflags --libs')
```

Comment:
Just for the reference (no need to change anything for now), the `libwslay.pc` pkgconfig file is not a given even on distros which package the library. wslay comes with two buildsystems, autotools and cmake, and only the former configures and installs the .pc file.

openSUSE use the autotools buildsystem and have thus `libwslay.pc` installed, so this will work.
Debian/Ubuntu use the CMake buildsystem, `<troll>`which, like any good CMake project, is CMake-centric and only installs its own `.cmake` config files`</troll>`. I'll open a ticket on https://salsa.debian.org/debian/wslay to suggest that they add a pkgconfig file.

For Mageia I used the CMake buildsystem, and wrote a `libwslay.pc` file myself. I use the same patch as Debian for the CMakeLists.txt to get a shared library with proper soname and `libwslay.so`, instead of `libwslay_shared.so` (ugh).